spec/graphql/relay/array_connection_spec.rb

```diff
@@ -157,13 +178,13 @@ def get_page_info(result)
 
       it "applies to queries by `last`" do
         last_cursor = "Ng=="
-        second_to_last_two_names = ["Death Star", "Shield Generator"]
+
         result = star_wars_query(query_string, "last" => 100, "before" => last_cursor)
-        assert_equal(second_to_last_two_names, get_names(result))
+        assert_equal(["Death Star", "Shield Generator"], get_names(result))
         assert_equal(true, get_page_info(result)["hasPreviousPage"])
 
         result = star_wars_query(query_string, "before" => last_cursor)
-        assert_equal(second_to_last_two_names, get_names(result))
+        assert_equal(["Yavin", "Echo Base"], get_names(result))
```

Comment:
Hmm, can you help me understand what's happening here? I _think_ the behavior of `"before"` without `"last"/"first"` is changing: 

- Previously, we implied an implicit `last: max_page_size`
- Now, we apply an implicit `first: max_page_size`

Is that correct? 

(Re-reading the spec, this may be correct, but I want to make sure to think through the compatibility issues in rolling this out.)